pimpmycause/profiles/models.py

```diff
@@ -1,26 +1,69 @@
 from __future__ import unicode_literals
 
 from django.db import models
-from django.contrib.auth.models import User
```

Comment:
Thanks again Lili, I'm gonna google a bit and see how this works, I don't know even how to see admin :/

It looks ok to me, but not sure if I'm the right person to upvote as I'm not familiar with this code. If no one else upvote and I manage to learn, I will come back to check and upvote.